docs/content/preview/admin/ysql-dumpall.md

```diff
@@ -137,23 +147,23 @@ Do not dump subscriptions.
 
 #### --no-sync
 
-By default, `ysql_dumpall` will wait for all files to be written safely to disk. This option causes `ysql_dumpall` to return without waiting, which is faster, but means that a subsequent operating system crash can leave the dump corrupt. Generally, this option is useful for testing but should not be used when dumping data from production installation.
+By default, ysql_dumpall waits for all files to be written safely to disk. This option causes ysql_dumpall to return without waiting, which is faster, but means that a subsequent operating system crash can leave the dump corrupt. Generally, this option is helpfull for testing but should not be used when dumping data from production installation.
```

Comment:
on line 128 , there s a note for --no-role-passwords in square brackets in bold. Format it.